change/@microsoft-fast-components-69608666-d48d-4a34-8135-4f275ac1788c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This should probably be a chore because updating the examples shouldn't cause a version-change for the package.

Reply:
hmm chore isn't a type, so none or patch are the best options.